AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -716,6 +716,8 @@ experts-exchange.com##.ctaSolutionStickyWrapper
 proshop.dk##.site-background-banner
 proshop.dk##a[data-action="campaignClick"][data-position="bottom-left"]
 proshop.dk##.site-top-banner
+byjus.com###top-banner-register
+byjus.com#?#.sidebar > div.bgc-white:has(> div.relatedlinks-widget > div.widget-content > a[target="_blank"])
```

Comment:
why not 
`byjus.com#?#.sidebar > div.bgc-white:has(> div.relatedlinks-widget > div.widget-content > a[href^="https://byjus.com/byjus-classes-book-a-free-demo-class/"])`

Reply:
@stanislav-atr

Reply:
They might change banner in the future, but `target` will most probably stay.

Reply:
`a[onclick*="sidebar register click"]`

Reply:
I swapped to `a[onclick*="sidebar register click"]` as Alex suggested, but i still think it might be changed in the future, seeing how other a's have different onclick `ga` arguments.